src/containers/BooksForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BookForm = () => (
+  <form>
+    <input placeholder="Book Title" type="text" />
+    <select>
+      <option>Action</option>
```

Comment:
- [ ] **[OPTIONAL]** What about having an array to store categories and map through it to generate the option tags? It would be more DRY.